admin_guide/limits.adoc

```diff
@@ -87,7 +87,36 @@ specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
 ====
-// end::admin_limits_sample_definition[]
+
+.OpenShift Limit Range Object Definition
+====
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "openshift-resource-limits"
+spec:
+  limits:
+    -
```

Comment:
> The thing is that giving this to a yaml validator, it will indent the object aka associative array in the same way you see here.

IIUC both styles (hyphen on its own line, hyphen on same line as object) are valid.  So, there's no urgent problem leaving it as-is, except for consistency w/ other YAML fragments in the docs.  However, sooner or later, someone will complain and we will change it, so might as well change it sooner than later...

> Moreover with squashed lines the ascii_binder will render everything following the dash in red making it look like this is invalid syntax.

I don't understand this part.  It sounds like you're talking about the `watch` feature (or bug, as it may be in this case).  I don't use that, so it doesn't bother me.  But if it bothers you, maybe you can [mention it](https://github.com/redhataccess/ascii_binder/issues) to @nhr.

> I'd have to remove the leading [source,yaml] line.

That's a step backwards, so i don't think it's a good idea.